app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -7,6 +7,10 @@ module SmartProxyExtensions
       PULP_NODE_FEATURE = "Pulp Node"
 
       included do
+        include ForemanTasks::Concerns::ActionSubject
+        include Katello::Glue::Pulp::Node if SETTINGS[:katello][:use_pulp]
+        include Katello::Glue if SETTINGS[:katello][:use_pulp]
```

Comment:
nitpick, its the same condition, might simply wrap it in an if
```ruby
if SETTINGS[:katello][:use_pulp]
  include Katello::Glue::Pulp::Node
  include Katello::Glue
end
```

Also, I do not know if the order matters, but I would put the top level module before, e.g.
```ruby
if SETTINGS[:katello][:use_pulp]
  include Katello::Glue
  include Katello::Glue::Pulp::Node
end
```